js/templates/pageNav.html

```diff
@@ -32,20 +32,25 @@
       </div>
       <div class="rowDivV colBorderBackground"></div>
       <div>
-        <div class="flexVCent">
+        <div class="flexVCent box">
           <a class="iconBtn ion-android-notifications"></a>
-          <a class="discSm border colBorder"
+          <a class="discSm border colBorder js-navListBtn"
           <% if (ob.navimage) { %>
           //this will be updated to the real image parameter
           style="background-image: url(ob.navImage), url('../imgs/defaultUser.png')"
           <% } else { %>
             style="background-image: url('../imgs/defaultUser.png')"
           <% } %>
           ></a>
+          <nav class="box border boxList navList colBorder colPrimary padKids shad2 foldDown js-navList">
```

Comment:
Not saying this is right or wrong... just expressing as a fellow developer that it just seems like a whole lot of classes for one little drop down. I'm all for abstraction, but it's just confusing to try and understand what is a 'box'... what is a 'boxList'... difference between that and 'navList'... how 'foldDown' changes things.

It just would be so much easier to have a drop-down component (with 1 main class and possibly a couple of modifiers) and be able to go to that sass file and see exactly what's going on.

You definitely don't want to keep repeating your CSS and making overly bulky css, but is there some type of middle ground to this approach?